lib/smack/SmackOptions.cpp

```diff
@@ -14,8 +14,12 @@ const llvm::cl::list<std::string> SmackOptions::EntryPoints(
   llvm::cl::value_desc("PROCS")
 );
 
-const llvm::cl::opt<bool> SmackOptions::Warnings(
-  "warnings", llvm::cl::desc("Enable warnings.")
+const llvm::cl::opt<std::string> SmackOptions::WarningType(
```

Comment:
What are the available warning types? I thought you can create an option with preset available types.